package.json

```diff
@@ -22,7 +22,7 @@
     "file-loader": "6.0.0",
     "flatpickr": "4.6.3",
     "immutability-helper": "3.0.2",
-    "jquery": "3.5.0",
+    "jquery": "^3.4.1",
```

Comment:
The `^` probably makes `npm` install `3.5.0` instead of `3.4.1` - maybe that's why it's still failing?

Reply:
nope, it has no error and it was broken before jquery update, it seems to be same as after the bootstrap update but i'm not sure why it's started again?! have removed caret now though 
:p